routes/web.php

```diff
@@ -16,5 +16,6 @@
 });
 Auth::routes();
 Route::get('/my_profile', 'HomeController@index')->name('my_profile');
+Route::get('/users', 'UserController@users');
```

Comment:
What is this for?

Reply:
I removed Code and
I removed UserController, which has no functional problem.

Reply:
it's still here.